docs/organizations/managing-security-and-risk.md

```diff
@@ -163,7 +163,16 @@ Codacy closes a finding when it detects that the associated Jira issue is marked
 
 Codacy opens a finding for each security issue detected during a penetration test.
 
-Codacy closes a finding when a subsequent penetration test does not detect the underlying security issue.
+Codacy closes a finding when a subsequent penetration test doesn't detect the underlying security issue.
+
+### How Codacy manages findings detected during application scanning (DAST) {: id="opening-and-closing-app-scanning-items"}
+
+!!! note
+    To view application scanning findings, also known as DAST (Dynamic Application Security Testing) findings, you must first [generate a DAST report and upload it to Codacy](../codacy-api/examples/uploading-dast-results.md).
+
+Codacy opens a finding for each security issue detected in the DAST report.
```

Comment:
We also update findings if subsequent reports have the same findings. Or re-open findings if subsequent reports find previously closed issues.

And more as seen here: https://codacy.slite.com/app/docs/D7nZEdFoHb3hif#1d2521bb

Not sure if we want to go into detail.

Reply:
Thanks. I think your remarks should be enough. We should document edge cases on a case-by-case basis (e.g. when someone complains)